style.scss

```diff
@@ -114,6 +114,8 @@
 
 @import "bower_components/scally/core/functions/string-replace";
 
+@import "core/functions/strip-unit";
```

Comment:
I know this isn't the best setup and this file will get blown away from the repo but for the sake of consistency can we follow the format of the all the other `@import` paths?